src/lib/accessibility/Behaviors/Input/InputBehavior.ts

```diff
@@ -0,0 +1,23 @@
+import { IAccessibilityBehavior, ComponentState } from '../../interfaces'
+
+export class InputBehavior implements IAccessibilityBehavior<{}, {}> {
+  private attributes = {
+    'ms-acc-behavior': this.name,
+  }
+
+  public get name(): string {
+    return 'input'
+  }
+
+  public generateAriaAttributes(props, state): object {
+    return this.attributes
+  }
+
+  public changeState(newState: ComponentState): void {
+    if (newState === ComponentState.disabled) {
+      this.attributes['aria-disabled'] = true
+    } else if (newState === ComponentState.enabled) {
+      delete this.attributes['aria-disabled']
```

Comment:
Seems `changeState` will not be required.  Let's just compute all new attributes on each render.